ui/branding/templates/header.html

```diff
@@ -1,28 +1,56 @@
-{% extends "oarepo_ui/header.html" %}
+{% extends "oarepo_ui/header.html" %} 
+
+{%- block navbar %}
+  <div class="ui container invenio-header-container">
+    <nav id="invenio-nav" class="ui menu shadowless borderless p-0">
+      
+      {# Main Site logo #}
+      {%- block navbar_header %}
+      <div class="item logo left-logo p-0">
+        {%- block brand %}
+          {%- if config.THEME_LOGO %}
```

Comment:
Use <language> trick mentioned in https://github.com/EOSC-CZ/nma/pull/16/files#r2022484583